app/src/types/Task.d.ts

```diff
@@ -17,6 +17,8 @@ export enum TaskPriority {
   Urgent = 'Urgent',
 }
 export interface ITask extends PartialBlock {
+  title: text
+  assignedTo: user
```

Comment:
Could you please just remove them, because we have the `title` and `assignedTo` nowhere 🥸